broker-cli/proto/broker.proto

```diff
@@ -1194,6 +1222,56 @@ service WalletService {
       body: "*"
     };
   }
+
+  /**
+   * CreateWallet creates a wallet within an engine. This RPC is used as during a 'first-use'
+   * setup.
+   *
+   * When a broker daemon is first getting setup, no crypto-wallets will exist on any of
+   * the provided engines. It is the user's responsiblity to create a wallet for each currency
+   * and to save the backup information (password and cipher seeds) that are used and returned
+   * from this method.
+   *
+   * ```javascript
+   * const { recoverySeed } = await walletService.withdrawFunds({ symbol: 'BTC', password: 'your-secure-password')
+   * console.log(recoverySeed)
+   * ```
+   *
+   * ```shell
+   * > sparkswap wallet create BTC
+   * ```
+   *
+   * > The above command will print:
+   *
+   * ```javascript
+   * > Please enter a password:
```

Comment:
This is what the javascript would do? That seems a bit weird looking at the code above

Reply:
Hah nope. good catch. This was bad docs